benchmarks/benchmark-echelon.C

```diff
@@ -120,9 +120,8 @@ typename Field::Element* M_randgen(const Field& F, typename Field::Element* L,ty
        FFLAS::ftrmm(F,  FFLAS::FflasRight, FFLAS::FflasUpper, FFLAS::FflasNoTrans, FFLAS::FflasNonUnit, m,n,1.0, U, lda, L, lda);
        */
 
-    const FFLAS::CuttingStrategy meth = FFLAS::RECURSIVE;
-    const FFLAS::StrategyParameter strat = FFLAS::THREE_D;
-    typename FFLAS::ParSeqHelper::Parallel pWH (MAX_THREADS, meth, strat);
+
+    FFLAS::ParSeqHelper::Parallel> pWH (MAX_THREADS);
```

Comment:
this file does not compile

Reply:
The same file from the master branch does not seem to compile due to the ParSeqHelper, maybe the implementation of ParSequHelper has been changed ever since ?

Reply:
Ok, I did not realize that this file was not in the benchmark suite. Forget it then!